### ISSUE TITLE
Provide mechanism for testing JSONSchema backwards compatibility

### ISSUE THREAD

**Issue by rafrombrc** [mozilla/schemaker#7]
It would be nice to provide a mechanism for folks to provide two versions of a JSONSchema and have the service return whether or not the newer version is backwards compatible with the earlier version.

**Comment by jainanup127** [comment]
yes that will be very helpful. Do you know any such library that already have such implementation
